timetracker/entries/templates/clients.html

```diff
@@ -0,0 +1,13 @@
+{% extends 'base.html' %}
+
+{% block title %}
+Clients
+{% endblock %}
+
+{% block clients %}
+      <ul>
+        {% for client in client_list %}
+        <li><a href="{% url 'client_summary' client.pk %}">{{ client.name }}</a></li>
```

Comment:
Since you called the `pk` parameter in the `urls.py` `pk` (`(?P<pk>\d+)`) you should / have to use the name in the url template tag too: `{% url 'client_summary' pk=client.pk %}`